### PR TITLE
Update Device endpoint docs

### PR DESCRIPTION
**Pull request by th3fallen**
Refs https://github.com/wheniwork/apitests/pull/7

### CODE REVIEW

File: source/methods/devices.md.erb [hunk 1]
```diff
@@ -10,6 +10,94 @@ Mobile apps use the POST endpoint to register a device and the DELETE endpoint t
...
+
+<% json do %>
+{
+  "device": [
+    {
+      "id": 1739,
+      "account_id": 1,
+      "user_id": 1,
+      "type": "android",
+      "token": "androidfolyfe",
```
Comment (bvisness) [inline source/methods/devices.md.erb]: @jswatkowski Do you think this is ok to have as a token example in the docs? I know we have "ilovemyboss" elsewhere...I guess I am asking how professional we want to be with this.

File: source/methods/devices.md.erb [hunk 2]
```diff
@@ -10,6 +10,94 @@ Mobile apps use the POST endpoint to register a device and the DELETE endpoint t
...
+    "id": 1739,
+    "account_id": 1,
+    "user_id": 1,
+    "type": "android",
+    "token": "androidfolyfe",
+    "created_at": "Fri, 25 Sep 2015 08:44:51 -0700",
+    "updated_at": "Fri, 25 Sep 2015 08:44:54 -0700"
+  }
+}
+<% end %>
```
Comment (bvisness) [inline source/methods/devices.md.erb]: You could use the `print_json` method here to drop in the example Device object from objects.json.

Example: https://github.com/wheniwork/wheniwork-api/blob/feature/update-devices-endpoints/source/methods/shifts.md.erb#L39-L43

**Comment by bvisness** [comment]
The formatting of this looks good, so once those other questions are answered I think we can merge this.

File: source/methods/devices.md.erb (hunk 2 above)
Comment (th3fallen) [inline source/methods/devices.md.erb]: Sorry im not following, is there some advantage to this method vs how i have it?

Comment (bvisness) [inline source/methods/devices.md.erb]: This way is probably ok, but `print_json` will use the `device` information from objects.json. That way if we ever want to change the example token or example ID, we can just change it in one place.

Comment (th3fallen) [inline source/methods/devices.md.erb]: Gotcha, updated.

**Pull request merged**